.github/skills/indicator-buffer/references/interface-selection.md

```diff
@@ -0,0 +1,111 @@
+# BufferList interface selection guide
+
+Use this reference to select the correct interface for BufferList implementations.
+
+## Interface decision tree
+
+```text
+Does indicator need OHLCV data (High, Low, Open, Close, Volume)?
+├─ Yes → Does it need TWO synchronized series?
+│  ├─ Yes → IIncrementFromPairs (Correlation, Beta)
+│  └─ No → IIncrementFromQuote (Stoch, ATR, VWAP, ADX)
```

Comment:
Technically all `BufferList` (and all styles) implementations support `IQuote` inputs from base class abstract, it’s just that some only support that mode.